### PR TITLE
Feature/interact with firebase

### PR DESCRIPTION
**Pull request by hotay**
- Modify firebase config including url and key path in `config.js`

``` javascript
...
FIREBASE_URL: 'https://mxfactorial.firebaseio.com',
FIREBASE_KEY_PATH: 'mxfactorial-cebe88ad95c6.json',
...
```
- Run server with `npm start` or `NODE_PATH=. node main`
#### Added `firebase-client` module:
- Login to google apis and generate jwt token apply this token to rest client header
- How to use:

``` javascript
var firebaseClient = require('firebase-client/index');

firebaseClient().then(function (instance) {
// run a request here using instance, instance is axios client instance
  return instance.get('/accounts'); // must return it to keep the `Promise`
}).then(function (response) {
// Handle respone from firebase
}).catch(function (err) {
// Handle error from the firebase
});
```
- For more examples visit `account/crud.js` and `transact/crud.js`

https://github.com/systemaccounting/mxfactorial/issues/43

### CODE REVIEW

**Comment by mxfactorial** [comment]
Bug 1:
Missing properties from account record:
country
city
state
industry
occupation
email_address

Minor UI change request: Anchor addGoodOrServiceButton button to above the transactButton instead of below the sumTransactionItemsDisplayCell (FYI @ngvtuan).

**Comment by hotay** [comment]
@mxfactorial I fix that bug, you can test now. And also minor tweak ui like your suggestion.

**Comment by mxfactorial** [comment]
Bug 1 unresolved: `street_name` & `state` missing from account record

New bug 2: cr_account missing for 1 of 2 transaction_item records:

missing-cr_account

File: firebase-client/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var google = require('googleapis');
+
+var config = require('config.js');
```
Comment (ghost) [inline firebase-client/index.js:3]: Hi this refers to [Note](https://github.com/systemaccounting/mxfactorial/pull/50/files#r70177293)

**Pull request merged**